mslib/mscolab/models.py

```diff
@@ -58,16 +58,19 @@ class User(db.Model):
     username = db.Column(db.String(255))
     emailid = db.Column(db.String(255), unique=True)
     password = db.Column(db.String(255))
+    profile_image_path = db.Column(db.String(255), nullable=True)  # relative path
```

Comment:
We definitely need that proper migration machinery and an automated check for model changes, then we would have noticed that this requires a migration... I am working on that already, so I think we can add that later, but it would mean that develop would temporarily not be in a releasable state.